sycl/test/regression/cuda_program_interop.cpp

```diff
@@ -0,0 +1,22 @@
+// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple %s -o %t.out
+// RUN: env SYCL_DEVICE_TYPE=GPU %t.out
+// REQUIRES: cuda
+
+#include <CL/sycl.hpp>
+#include <CL/sycl/backend/cuda.hpp>
+
+int main() {
+  sycl::queue Queue{sycl::default_selector{}};
+  Queue.submit(
+      [](sycl::handler &CGH) { CGH.single_task<class TestKernel>([] {}); });
+
+  sycl::program Prog{Queue.get_context()};
+
+  auto NativeProgram = Prog.get_native<sycl::backend::cuda>();
+
```

Comment:
Could you please also create a program from the native handle to test piextProgramCreateWithNativeHandle ?

Reply:
I'd love to, but I only see implementations of `make_platform` for L0 and OpenCL backends, not for CUDA. That's probably why we haven't face that issue before. I only found this by staring at the code.

Reply:
Ok.